server/configuration.go

```diff
@@ -25,6 +25,7 @@ func (p *MatterpollPlugin) OnConfigurationChange() error {
 		return err
 	}
 
+	p.ServerConfig = p.API.GetConfig()
```

Comment:
I didn't check for `nil`consciously. If fetching `GetConfig()` fails, what do we do?

I'm quite sure this will never be a problem.

Reply:
Agree. If `GetConfig()` is failed, it's not sure that other plugin functions work correctly too. Matterpoll can do nothing here.